emailconfirmation.py

```diff
@@ -0,0 +1,37 @@
+from main import app
+from itsdangerous import URLSafeTimedSerializer
+from flask_mail import Mail, Message
+from pyclub.dbconnect import get_user
+
+mail = Mail(app)
+
+def generate_confirmation_token(email):
+    serializer = URLSafeTimedSerializer("Hello there")
+    return serializer.dumps(email, salt="General Kenobi")
+
+
+def confirm_token(token):
+    serializer = URLSafeTimedSerializer("Hello there")
```

Comment:
Nie dałoby się tego zrobić lepiej? [o to co](https://github.com/PRD-ev/pyclub/blob/57acbc881f38744bc83e9d6fb8b3f50ccce4210b/emailconfirmation.py#L9)

Reply:
jak lepiej ?

Reply:
Jak dla mnie jest całkiem dobrze